### ISSUE TITLE
missing tree_stats_table.csv in HybPhyloMaker6a2_RAxML_trees_summary

### ISSUE THREAD

**Issue by biomendi** [tomas-fer/HybPhyloMaker#11]
Hi Tomas,

I successfully ran `HybPhyloMaker6a_RAxML_for_selected` using a large set of uncorrected alignments, which I partitioned by exon (genetreepart=exon). 

However, I noticed an issue with `HybPhyloMaker6a2_RAxML_trees_summary` as it did not find the `tree_stats_table.csv` file. 

Here the error:

```
paste: tree_stats_table.csv: No such file or directory
cp: cannot stat '*.png': No such file or directory
cp: cannot stat 'genes_corrs.*': No such file or directory
rm: cannot remove 'genes_corrs.*': No such file or directory

```

If possible, I would really appreciate your help. 

Thanks a lot!

The R.log 

```
Calculating tree properties using tree_props.r


Attaching package: ‘seqinr’

The following objects are masked from ‘package:ape’:

    as.alignment, consensus

[1] 29.44999
[1] 40.29817
[1] 132.2465
.
.
.
[1] 28.66668
[1] 29.21471
[1] 47.54456
Warning message:
In cbind(paste(aln_dir, aln_files, sep = ""), paste(trees_dir, trees_files,  :
  number of rows of result is not a multiple of vector length (arg 2)
Error in model.frame.default(formula = p_dist ~ branch_dist, drop.unused.levels = TRUE) : 
  variable lengths differ (found for 'branch_dist')
Calls: t ... lm -> eval -> eval -> <Anonymous> -> model.frame.default
Execution halted

Calculating and parsing LB score using LBscores.R


Attaching package: ‘seqinr’

The following objects are masked from ‘package:ape’:

    as.alignment, consensus

There were 50 or more warnings (use warnings() to see the first 50)

Plotting boxplots/histograms for tree properties using treepropsPlot.r

Error in read.table(file = file, header = header, sep = sep, quote = quote,  : 
  no lines available in input
Calls: read.csv -> read.table
Execution halted

Plotting gene properties correlations for RAxML trees using plotting_correlations.R

Error in eval(substitute(select), nl, parent.frame()) : 
  object 'Bootstrap' not found
Calls: subset -> subset.data.frame -> eval -> eval
Execution halted
```

**Comment by tomas-fer** [comment]
Hi, it looks like you do not have the same number of selected alignments and trees. Could you check this, please?

**Comment by biomendi** [comment]
Solved, thank you!

It turns out that RAxML did not calculate the trees for some alignments that had a lot of missing data. I removed those alignments from the 'selected_genes.txt' file. 

After that, I also encountered some problems with the script `treepropsPlot.r` due to the presence of NAs in the column 'Slope' of 'tree_stats_table.csv'.  I fixed that by adding `x[is.na(x)] <- 0`.

Finally, I found out that the problematic alignments must also be removed from the file 'summarySELECTED_99_1.txt'. Otherwise, the function read.table() in the script 'tree_props.r' will produce error due to those problematic alignments not having the same number of rows.
